### PR TITLE
[MI-2511] Created constants for common mocks

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**

### CODE REVIEW

File: server/plugin/api_test.go [hunk 1]
```diff
@@ -314,17 +314,17 @@ func TestHandleDeleteAllSubscriptions(t *testing.T) {
 	}{
 		{
 			description: "HandleDeleteAllSubscriptions: valid",
-			userID:      "mockMattermostUserID",
-			projectID:   "mockProjectID",
+			userID:      testutils.MockMattermostUserID,
+			projectID:   testutils.MockProjectID,
 			statusCode:  http.StatusOK,
 			subscriptionList: []*serializers.SubscriptionDetails{
```
Comment (ayusht2810) [inline server/plugin/api_test.go:320]: use `testutils.GetSuscriptionDetailsPayload`

File: server/plugin/api_test.go [hunk 2]
```diff
@@ -731,14 +731,14 @@ func TestHandleGetSubscriptions(t *testing.T) {
...
-			marshalError:  errors.New("mockError"),
+			marshalError:  errors.New("error while marshaling"),
 			statusCode:    http.StatusInternalServerError,
 			isTeamIDValid: true,
 		},
 		{
 			description: "HandleGetSubscriptions: GetSubscriptionsForAccessibleChannelsOrProjects gives error",
 			project:     testutils.MockProjectName,
-			GetSubscriptionsForAccessibleChannelsOrProjectsError: errors.New("mockError"),
+			GetSubscriptionsForAccessibleChannelsOrProjectsError: errors.New("error while getting subscriptions for accessible cheannels or project"),
```
Comment (ayusht2810) [inline server/plugin/api_test.go:741]:
```suggestion
			GetSubscriptionsForAccessibleChannelsOrProjectsError: errors.New("error while getting subscriptions for accessible channels or projects"),
```

File: server/plugin/api_test.go [hunk 3]
```diff
@@ -810,14 +803,14 @@ func TestHandleSubscriptionNotifications(t *testing.T) {
...
-			err:         errors.New("mockError"),
+			err:         errors.New("error empty body"),
 			channelID:   "mockChannelIDmockChannelID",
 			statusCode:  http.StatusOK,
 		},
 		{
 			description: "SubscriptionNotifications: invalid channel ID",
 			body:        `{}`,
-			err:         errors.New("mockError"),
+			err:         errors.New("error invalid channel id"),
```
Comment (ayusht2810) [inline server/plugin/api_test.go:813]:
```suggestion
			err:         errors.New("error invalid channel ID"),
```

File: server/plugin/command_test.go [hunk 4]
```diff
@@ -233,8 +233,8 @@ func TestExecuteCommand(t *testing.T) {
 			if testCase.isListCommand || testCase.isDeleteCommand {
 				mockedStore.EXPECT().GetAllSubscriptions("").Return([]*serializers.SubscriptionDetails{
```
Comment (ayusht2810) [inline server/plugin/command_test.go:234]: same

File: server/plugin/utils_test.go [hunk 5]
```diff
@@ -216,20 +216,20 @@ func TestEncrypt(t *testing.T) {
...
-			newGCMError:   errors.New("mockError"),
+			expectedError: "newGCMError",
+			newGCMError:   errors.New("newGCMError"),
 		},
 		{
 			description:   "Encrypt: io.ReadFull give error",
 			secret:        "mockSecret",
-			expectedError: "mockError",
-			resdFullError: errors.New("mockError"),
+			expectedError: "resdFullError",
```
Comment (ayusht2810) [inline server/plugin/utils_test.go:231]:
```suggestion
			expectedError: "readFullError",
```

File: server/plugin/utils_test.go [hunk 6]
```diff
@@ -515,19 +515,19 @@ func TestIsSubscriptionPresent(t *testing.T) {
 			description: "test IsSubscriptionPresent with subscription present in subscription list",
 			subscriptionList: []*serializers.SubscriptionDetails{
```
Comment (ayusht2810) [inline server/plugin/utils_test.go:516]: same. check everywhere

**Review by ayusht2810 — APPROVED** [review]

**Review by Kshitij-Katiyar — APPROVED** [review]

**Pull request merged**